x-pack/qa/rolling-upgrade/src/test/resources/rest-api-spec/test/mixed_cluster/110_enrich.yml

```diff
@@ -0,0 +1,13 @@
+---
+"Enrich stats query smoke test for mixed cluster":
+  - skip:
+      version: " - 7.99.99"
```

Comment:
Actually, is this right? This is the 7.9 backport.

Reply:
A silly error of cherry pick and merge... This file (`110_enrich.yml`) is redundant and the correct one is `120_enrich.yml`. I replaced it with the content of `120_enrich.yml`.